types/react-native-web/README.md

```diff
@@ -0,0 +1,14 @@
+# `react-native-web`
+
+> React Native for Web is an accessible implementation of React Native's Components and APIs that is interoperable with React DOM. https://necolas.github.io/react-native-web/
+Because it is not an identical implementation of `react-native`, it adds or modifies existing type signatures on exported components and APIs from `react-native`.
+
+These types are an attempt to augment existing `react-native` types to match the behavior when `react-native-web` is implemented in a project.
+
+## Usage 
+
+To enhance `react-native` types, users must import these specific types into any `.ts` or `.d.ts` file. By doing this, they are augmenting the underlying `react-native` types with web specific properties.
```

Comment:
I think we can't import like this `import '@types/react-native-web';` in `.ts` files that are used by the RN application -- it lead to runtime crashes when I did it with previous PR, unless I add the extension: `import '@types/react-native-web.d.ts';`

Reply:
Turns out we can import `react-native-web`, and it works magically 🪄 

Updated the usage